mkc.json

```diff
@@ -2,7 +2,14 @@
     "targetWebsite": "https://maker.makecode.com",
     "links": {
         "jacdac": ".",
+        "jacdac-accelerometer": "./accelerometer",
```

Comment:
it only makes sense to add this, if we have other packages that depend on them; do we?

Reply:
removed